src/components/SearchForm/_search-form.scss

```diff
@@ -0,0 +1,53 @@
+// ----------------------------------
+// UI Components
+// Search Form
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.search-form {
+  border: 1px solid var(--colour-concrete-10);
+  margin-bottom: rem(30);
```

Comment:
Lots of rules containing `rem(<value>)`. These should be replaced with rounded multiples of --space-unit wherever possible. Happy for this to happen later but all tech debt needs ticketing

Reply:
Imagine this will cause inconsistencies in the look of components as we port their styles from https://github.com/wellcometrust/corporate, though?

Reply:
Minor changes at most

By rounding I mean to numbers which are relatively clean multiples of 8 to 2 decimal places :)